kernel/sorts.ml

```diff
@@ -463,12 +454,16 @@ let debug_print = function
   | QSort (q, u) -> Pp.(str "QSort(" ++ QVar.raw_pr q ++ str ","
                         ++ spc() ++ Univ.Universe.raw_pr u ++ str ")")
 
-let pr_sort_family = function
-  | InSProp -> Pp.(str "SProp")
-  | InProp -> Pp.(str "Prop")
-  | InSet -> Pp.(str "Set")
-  | InType -> Pp.(str "Type")
-  | InQSort -> Pp.(str "Type") (* FIXME? *)
+let pr prv = function
+  | SProp -> Pp.(str "SProp")
+  | Prop -> Pp.(str "Prop")
+  | Set -> Pp.(str "Set")
+  | Type u -> Pp.(str "Type@{" ++ Univ.Universe.raw_pr u ++ str "}")
```

Comment:
you also need to take a univ level printer in this function, then you can call Universe.pr here